makePoplogCommander.sh

```diff
@@ -938,12 +938,14 @@ cat << \****
         deque_pop_front( argd );
         return processArgs( argd, base, flags, envv );
     } else {
-        fprintf( stderr, "Unexpected arguments:" );
-        for ( int i = 0; i < deque_length( argd ); i++ ) {
-            fprintf( stderr, " %s", (char *)deque_get( argd, i ) );
-        }
-        fprintf( stderr, "\n" );
-        return EXIT_FAILURE;
+        setUpEnvironment( base, flags, envv );
+        char * subpath = "/pop/getpoploglib/lib/getpoplog_run_subcommand.p";
+        char * path = safe_malloc( strlen( subpath ) + strlen( base ) + 1 );
+        char * p = stpcpy( path, base );
+        p = stpcpy( p, subpath );
+        deque_push_front( argd, path );
+        deque_push_front( argd, "pop11" );
+        execvp( "pop11", deque_as_array( argd ) );
```

Comment:
Should we not interrogate the return value and display an error to the user? Or is the idea that running pop11 will display that error message?

Shouldn't we exit with a non-zero exit code if this call returns a non-zero exit code?

Reply:
Pop11 will return a suitable exit code & error message. I will add a comment to that effect.